client/helpers.go

```diff
@@ -190,9 +190,32 @@ func applyRootRoleChange(repo *tuf.Repo, c changelist.Change) error {
 	return nil
 }
 
-func nearExpiry(r *data.SignedRoot) bool {
+func nearExpiry(r data.SignedCommon) bool {
 	plus6mo := time.Now().AddDate(0, 6, 0)
-	return r.Signed.Expires.Before(plus6mo)
+	return r.Expires.Before(plus6mo)
+}
+
+func rolesNearExpiry(r *tuf.Repo) error {
```

Comment:
can we rename this function to describe the warning?  Something like `warnRolesNearExpiry`?